python/ray/train/lightning/lightning_checkpoint.py

```diff
@@ -0,0 +1,61 @@
+from typing import TYPE_CHECKING, Optional, Type, Union, Dict
+
+from ray.air.checkpoint import Checkpoint
+from ray.util.annotations import PublicAPI
+
+if TYPE_CHECKING:
+    from pytorch_lightning import LightningModule
+    from ray.data.preprocessor import Preprocessor
+
+
+@PublicAPI(stability="alpha")
+class LightningCheckpoint(Checkpoint):
+    """A :py:class:`~ray.air.checkpoint.Checkpoint` with Lightning-specific
+    functionality.
```

Comment:
First line of docstring should be 1 line please! https://peps.python.org/pep-0257/#multi-line-docstrings